### DISCUSSION TITLE
Doesn't seem to work out of box with NextJS in a monorepo

### DISCUSSION THREAD

**Discussion by osdiab** [chakra-ui/panda#847] [Q&A] [answered]
### Description

I added Panda CSS to a NextJS app in a monorepo. Code for Panda is only written in the monorepo, but styles don't seem to be applying consistently. Sometimes I could get individual `css()` invocations to show up, I can never get patterns like `hstack()` to work.

### Link to Reproduction

https://github.com/osdiab/nextjs-panda-monorepo/commit/20a2e3311396c3d1f3a7c08d1a873b6a18fe6df0

### Steps to reproduce

1. create turbo monorepo with pnpm
2. follow the panda setup instructions in the `apps/web` folder
3. add some styles to the `page.tsx` page
4. run the dev app

then you'll see

- the classnames show up
- the reset styles, etc are showing up
- the actual styles that you insert don't show up

### JS Framework

NextJS with Turborepo/pnpm workspaces

### Panda CSS Version

0.5.0

### Browser

Chrome

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment, marked as answer]
The configured path in the "include" option is incorrect that's why. Update it and it should work correctly.

![CleanShot 2023-06-27 at 08 44 33@2x](https://github.com/chakra-ui/panda/assets/6916170/476b19fd-90ac-4e83-a1a0-362bbf37db5b)

**Reply by osdiab** [reply]
You're right, that fixed it here. Thanks! So my repro case doesn't have the issue, still not sure why when I try to do this in another test project that already uses vanilla-extract that it seems to fail.

https://github.com/osdiab/personal-website/pull/11

So basically i get this markup:

<img width="497" alt="Screenshot 2023-06-28 at 14 44 55" src="https://github.com/chakra-ui/panda/assets/1890926/4aa8fefa-232d-4072-8f26-5e179502d606">

but the `hstack` fails to show up like so:

<img width="437" alt="Screenshot 2023-06-28 at 14 44 48" src="https://github.com/chakra-ui/panda/assets/1890926/e999965d-828f-44c6-b972-c1b935f93c91">

and even eradicating all traces of vanilla extract doesn't help: https://github.com/osdiab/personal-website/pull/12

<img width="157" alt="Screenshot 2023-06-28 at 14 54 11" src="https://github.com/chakra-ui/panda/assets/1890926/3bf6fea5-e852-40b1-923b-65d4bae9a3f2">

Sorry to be asking for support here but I'd really like to properly try out panda, but at a loss for what i'm doing wrong.
